plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -65,13 +75,13 @@ public boolean validatePage() {
   private boolean validateLocalFields() {
     String packageName = javaPackageField.getText();
     if (!JavaPackageValidator.validate(packageName)) {
-      setErrorMessage("Illegal Java package name: " + packageName);
+      setErrorMessage("Illegal Java package name: " + packageName); //$NON-NLS-1$
       return false;
     }
     
     String projectId = projectIdField.getText();
     if (!AppEngineProjectIdValidator.validate(projectId)) {
-      setErrorMessage("Illegal App Engine Project ID: " + projectId);
+      setErrorMessage("Illegal App Engine Project ID: " + projectId); //$NON-NLS-1$
```

Comment:
I know it's not directly part of this change, but it might be useful to describe what is a valid app id.  Maybe a tooltip on the description text.